content/enterprise/before-installing/reference-architecture/gcp.mdx

```diff
@@ -36,10 +36,11 @@ architecture.
 -> **Note:** This reference architecture focuses on the _External Services_ operational mode.
 
 Depending on the chosen [operational
-mode](/enterprise/before-installing/#operational-mode-decision),
-the infrastructure requirements for Terraform Enterprise range from a single Cloud Compute VM instance
-for demo installations to multiple instances connected to Cloud SQL and Cloud Storage for a
-stateless production installation.
+mode](/enterprise/before-installing/#operational-mode-decision), the
+infrastructure requirements for Terraform Enterprise range from a single Cloud
+Compute VM instance for _Mounted Disk_ installations to multiple instances
+connected to Cloud SQL and Cloud Storage for a stateless production
+installation.
```

Comment:
Again:

"The requirements depend on the operational mode. For example, an installation on Mounted Disk mode may require a single Cloud Compute VM instance, whereas a stateless production installation may require multiple instances connected to Cloud SQL and Cloud Storage.